docs/src/development/cli/_index.md

```diff
@@ -18,51 +18,42 @@ Its source code is hosted on [GitHub](https://github.com/platformsh/platformsh-c
 
 You need to have:
 
-* PHP 5.5.9+
-* Git
+* [PHP 5.5.9+](https://www.php.net/manual/en/install.php) with the following extensions installed: `curl`, `json`, `mbstring`, `pcre`, and `phar`
+* [Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
 * A Bash-like shell
 
-  On Windows, the best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
-  You can also use another Bash-compatible shell such as [Git Bash](https://gitforwindows.org/),
-  which you might have installed with git.
-
-For full requirements, see the [requirements on GitHub](https://github.com/platformsh/platformsh-cli#user-content-requirements).
+On Windows, the best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
 
 ## 1. Install
 
-Install the CLI using this command:
-
-```bash
-curl -fsS https://platform.sh/cli/installer | php
-```
-
-If you run into issues or want to install manually, see the [installation instructions on GitHub](https://github.com/platformsh/platformsh-cli#user-content-installation).
+{{% cli-installation %}}
 
 ## 2. Authenticate
 
-To see and manage projects, you need to be authenticated with Platform.sh
-Run this command:
+To see and manage projects, authenticate by running the following command:
 
 ```bash
-platform
+platform login
 ```
 
-You are asked to log in via a browser.
-This process creates SSH certificates on your computer for secure connections.
-(See more about [SSH connections](/development/ssh/_index.md).)
+This process opens a browser tab for you to log in.
+It also creates certificates on your computer for [SSH connections](/development/ssh/_index.md).
 
 Once you are logged in, a list of your projects appears, along with some tips for getting started.
```

Comment:
:lady_beetle: By changing the command to `platform login`, this last line becomes untrue. The list of projects appears when `platform` is run. So I guess going back to `platform` would be best (unless you want to change this line).

If `platform` is used, it includes this at the end
```
To view all commands, run: platform list
```
So then the same thing could be removed here.